admin/sections/config/providers/groq.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  FormControl,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+import { PasswordInput } from "@/components/ui/password-input";
+import { UseFormReturn } from "react-hook-form";
+import { ModelForm } from "./shared";
+
+export const GroqForm = ({
+  form,
+  defaultValues,
+}: {
+  form: UseFormReturn;
+  defaultValues: any;
+}) => {
+  const supportingModels = ["llama3-8b-8192", "llama3-70b-8192", "mixtral-8x7b-32768"];
```

Comment:
pretty sure that should be
```suggestion
  const supportingModels = ["llama3-8b", "llama3-70b", "mixtral-8x7b"];
```